### PR TITLE
Update labels

### PR DESCRIPTION
**Pull request by jacobtomlinson**
This PR adds the labels proposed in dask/community#50 and will sync them with dask/dask and dask/distributed.

I will need to manually rename existing labels which are remaining, otherwise they will be deleted and recreated resulting in issues and PRs losing their labels.

So I'm just looking for a couple of approvals here to go ahead rather than a merge.

cc @mrocklin @jsignell @martindurant @jcrist @jakirkham @jrbourbeau

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -1,71 +1,107 @@
...
-  description: ""
+  description: "Awaiting triage"
+
+- name: Type:Needs Attention
+  color: e58b7e
+  description: "It's been a while since this was pushed on"
+
+- name: Type:Needs Info
+  color: bebaf4
+  description: "Needs information further information"
```
Comment (jrbourbeau) [inline .github/labels.yml]:
```suggestion
  description: "Needs further information"
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 name: Bug Report
 about: Report a bug
 labels:
-  - type/bug
-  - needs-triage
+  - Type:Bug
+  - Type:Needs Triage
```
Comment (jrbourbeau) [inline .github/ISSUE_TEMPLATE/bug-report.md:6]: Looking forward to having "Needs Triage" applied automatically : )

**Review by jrbourbeau — APPROVED** [review]
Thanks @jacobtomlinson!

File: .github/labels.yml [hunk 3]
```diff
@@ -1,71 +1,107 @@
...
+  color: 0052cc
+  description: "Something is missing"
+
+- name: Type:Hygiene
+  color: bebaf4
+  description: "Code quality and project level things"
+
+- name: Type:Testing/CI
+  color: f4e49a
+  description: "Unit tests and/or continuous integration"
```
Comment (jsignell) [inline .github/labels.yml:32]: Should there be a `Type:Discussion` that doesn't get hit with the stale bot? We had discussed having an "active" label, but I think it is generally easier to label the less common case.

Comment (jacobtomlinson) [reply .github/labels.yml:32]: Added!

**Comment by jakirkham** [comment]
Is it possible for users to select the issue type so as to label it correctly? Thinking of what we have in RAPIDS. Admittedly I am not sure whether that is something we want to do here. Just wondering 🙂

**Comment by jrbourbeau** [comment]
When opening a new issue, one will have to pick between "Bug Report" or "Feature Request" (https://github.com/dask/dask/issues/new/choose), and the "Type:Bug" or "Type:Feature" label will be applied automatically. Is that what you're referring to, or do you mean other labels like "Area:Daraframe"?

**Comment by jakirkham** [comment]
Yeah that's what I was thinking. This came up in issue ( https://github.com/dask/dask/issues/4477 ) before. It might help cutdown on the maintenance effort. Though it does increase the effort for new users. Understand going either way here. So this is just a thought.

**Comment by jsignell** [comment]
AFAICT only people with triage access can add labels, so it is not possible for most people.

**Comment by jakirkham** [comment]
Right so the idea would be to bake this into the template where labels can be added even if the user has no permissions.

Edit: Again this is just an idea. Not saying we should or should not do it.

**Comment by jacobtomlinson** [comment]
This change means folks will be able to assign either the bug or enhancement labels by choosing the appropriate template. I think adding any further labels such as Area or Priority should be left down to the czar.

So I think this does satisfy your request @jakirkham.

**Review by jsignell — APPROVED** [review]
I am happy with this as is :)

**Comment by jsignell** [comment]
@jacobtomlinson do you think we should merge this?

**Comment by jacobtomlinson** [comment]
Sure. I'm a little apprehensive given the age of it. And I'm going to be out for the next couple of days. 

If you're happy to watch out for any fallout then go for it, otherwise, maybe wait until next week?

**Comment by jsignell** [comment]
yeah we can wait until next week :)

**Comment by jsignell** [comment]
I think we shoudl just go ahead and merge this and we can always tweak it as we implement labeling

**Review by fjetter — APPROVED** [review]

**Comment by jsignell** [comment]
Well that was anticlimactic :) it looks like this is depending on a branch that doesn't exist anymore, @jacobtomlinson do you by any chance remember what that was about?

**Comment by jsignell** [comment]
aha! Got it https://github.com/micnncim/action-label-syncer/pull/43 I'll open a PR to fix this up.

**Pull request merged**